types/remarkable/lib/index.d.ts

```diff
@@ -390,81 +401,430 @@ declare namespace Remarkable {
          */
         content: any;
 
-        /**
-         * The type of the token.
-         */
-        type: string;
-
         /**
          * Is this a block element
          */
         block: boolean;
     }
 
-    interface MiscTokenProps {
-        /**
-         * Image alt.
-         */
-        alt?: string;
+    // ---------------
+    // Specific Block Tokens
```

Comment:
Wow, you have added most rule token interfaces 👍 Could attach some links to related resources about the tokens and provide a few small tests?

Reply:
IMHO, it's still very perfectible.

For source I've basically read the source code: https://github.com/jonschlinkert/remarkable/tree/master/lib/rules_block when the token stack is updated.

I will add tests for that asap.